src/status_im/contexts/wallet/add_account/create_account/view.cljs

```diff
@@ -167,38 +168,59 @@
                                            :container-style     (style/slide-button-container bottom))]}]
        children))))
 
+(defn- on-auth-success--mnemonic
```

Comment:
why `--`? I think we should just keep the same format as always, no?

Reply:
should these not just be 
on-import-private-key-success, 
and on-mnemonic-success